client-js/evomaster-client-js/src/instrumentation/babel-plugin-evomaster.ts

```diff
@@ -190,6 +225,67 @@ export default function evomasterPlugin(
         branchCounter++;
     }
 
+    function replaceConditionalExpression(path: NodePath){
+
+        if(!t.isConditionalExpression(path.node)){
+            throw Error("Node is not a ConditionalExpression: " + path.node);
+        }
+        const exp = path.node as ConditionalExpression;
+
+        if(! exp.loc){
+            return;
+        }
+
+        const l = exp.loc.start.line;
+        /*
+            test ? consequent : alternate
+            test: Expression;
+            consequent: Expression;
+            alternate: Expression;
+
+            transformed code:
+                test? __EM__.ternary(()=>consequent, ... ) : __EM__.ternary(()=>alternate, ...)
+
+            here, we create additional two statements targets for 'consequent' and 'alternate'
+            for the statement targets,
+                if consequent(/alternate) is executed without exception, h is 1
+                otherwise h is 0.5
+
+            Note that we do not further replace 'test' here.
+            if it is related to condition, it will be replaced by other existing replacement and
+            additional branch will be added there.
+
+            From trello
+            where ternary needs to create 2 objectives:
+                - 1 for for when it is executed/called  (h=1)
+                - 1 for when no exception (h=0.5 and then h=1 if and onyl if () => B did not throw exception)
+            TODO: check with Andrea, I am not clear about the second point,
+                do we need to handle 'consequent' and 'alternate' differently?
+                in addition, do we need to handle 'throw exception' with different h?
+                how about itself is a throw exception expression, e.g., A? B: (throw e)
+                if needed, we might need to add a variable for a check with 't.isThrowStatement()'
+         */
```

Comment:
very interesting point! didn't think about it.
if it is a `throw` statement, can have `h=1`. can u add a test for it?